_includes/title.html

```diff
@@ -19,10 +19,14 @@
     <h1>
         {{ page.title | xml_escape }}
         <small>{{ page.subtitle }}</small>
+        
         {% if edit_path %}
             <a class="btn btn-primary pull-right" href="{{ edit_path }}">Edit on Github</a>
         {% elsif show_news_link %}
             <a class="btn btn-primary pull-right" href="/news/atom.xml">Subscribe via Atom</a>
         {% endif %}
     </h1>
+    {% if page.author %}
```

Comment:
This title is included on every page, so this will have unintended consequences. Info like this should go in the page- or category-specific templates.

Separately, I'm not sure that we want a visible author tag on tutorials. While attribution on the project pages is pointing out whose work it is (because they still own the rights to it), in a tutorial as I see it the purpose is more to tell future contributors who to talk to if they have questions. I think that previous discussions that we have had concluded that content everywhere but the project pages was the property of the ev3dev organization and not the original author(s).

This is just my opinion and it's not a strong one, so I think @dlech will have to make a decision that we stick with from now on.